components/Descriptions/src/index.svelte

```diff
@@ -0,0 +1,85 @@
+<script lang="ts">
+	import { setContext } from 'svelte';
+	import type {
+		DescriptionsItemColGroup,
+		DescriptionsItemLabelMap,
+		KDescriptionsCtx,
+		KDescriptionsProps
+	} from './types';
+	import {
+		descriptionsKey,
+		getPrefixCls,
+		type ReactiveContext,
+		reactiveContext
+	} from '@ikun-ui/utils';
+	import { clsx } from 'clsx';
+
+	export let title: KDescriptionsProps['title'] = '';
+	export let extra: KDescriptionsProps['extra'] = '';
+	export let border: KDescriptionsProps['border'] = false;
+	export let column: KDescriptionsProps['column'] = 3;
+	export let direction: KDescriptionsProps['direction'] = 'horizontal';
+	export let size: KDescriptionsProps['size'] = 'md';
+	export let cls: KDescriptionsProps['cls'] = undefined;
+	export let attrs: KDescriptionsProps['attrs'] = {};
+
+	let colLabelRefs: DescriptionsItemLabelMap = {};
+	let colGroup: DescriptionsItemColGroup = new Array(column).fill([]);
+	const ctxInstance = reactiveContext<KDescriptionsCtx>({
+		props: {
+			column,
+			border,
+			direction,
+			size,
+			colLabelRefs,
+			colGroup
+		}
+	});
+
+	setContext<ReactiveContext<KDescriptionsCtx>>(descriptionsKey, ctxInstance);
+
+	$: {
+		ctxInstance.evtMap.forEach((cb) => {
```

Comment:
You can add unit tests here to test whether KDescriptionsItem will synchronize props for updates.

Reply:
> You can add unit tests here to test whether KDescriptionsItem will synchronize props for updates.

I refactored them.